packages/jellyfish-crypto/src/elliptic.ts

```diff
@@ -14,7 +23,7 @@ export interface EllipticPair {
   /**
    * @return {Promise<Buffer>} compressed public key
    */
-  publicKey: () => Promise<Buffer>
+  publicKey: () => Promise<PublicKey>
```

Comment:
```suggestion
  publicKey: () => Promise<Buffer>

  publicKeyCompressed: () => Promise<Buffer>,
  
  publicKeyUncompressed: () => Promise<Buffer>,
```

Instead of breaking compatibility, add 2 new methods so we don't have to change the shape or how this is consumed?

Reply:
yea it was the pre-impl.. refactor to `Promise<PublicKey>` due to https://github.com/BirthdayResearch/jellyfishsdk/pull/2105/commits/698ef6b6d52c1f732ed86517cf72d85520772feb

hmm.. what about this??
```diff
-- publicKey: () => Promise<Buffer>
++ publicKeyV2: () => Promise<PublicKey>
```

Reply:
@fuxingloh  dropped the "refactored".. lighter now